### PR TITLE
Features/539591 restructure objects to allow agents app

### PR DESCRIPTION
**Pull request by nikolakukrika**
#### Work Item(s) 
Fixes [AB#539591](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/539591)

### CODE REVIEW

File: src/System Application/App/Security Groups/src/InheritedPermissionSetsPart.Page.al [hunk 1]
```diff
@@ -0,0 +1,116 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Security.AccessControl;
+
+page 9821 "Inherited Permission Sets Part"
```
Comment (grobyns) [inline src/System Application/App/Security Groups/src/InheritedPermissionSetsPart.Page.al:8]: do the app.json ranges need to be updated?

**Comment by grobyns** [comment]
Is there no update to test code needed?

**Review by grobyns — APPROVED** [review]

File: src/System Application/App/Security Groups/src/InheritedPermissionSetsPart.Page.al (hunk 1 above)
Comment (nikolakukrika) [reply src/System Application/App/Security Groups/src/InheritedPermissionSetsPart.Page.al:8]: Done thanks

**Comment by nikolakukrika** [comment]
> Is there no update to test code needed?
We cannot test this in an easy way as it requires users. The tests are not running with multiple users. This PR is only moving the code from Base App to System modules so CC will stay the same.

**Review by stkillen — APPROVED** [review]

**Review by Groenbech96 — COMMENTED** [review]
LGTM

**Review by Groenbech96 — APPROVED** [review]
LGTM!

**Review by JesperSchulz — APPROVED** [review]

**Pull request merged**